code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -415,13 +415,13 @@
 				. = TRUE
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/CtrlClick(mob/user)
-	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK) && !state_open)
+	if(can_interact(user) && !state_open)
 		on = !on
 		update_icon()
 	return ..()
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/AltClick(mob/user)
-	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+	if(can_interact(user))

```

Comment:
undocumented change, this is not really atmos equipment and may have balance concerns

Reply:
Came up in the search of the atmos folders so did them as well, this can be done through the ui anyway.